index.html

```diff
@@ -436,30 +451,39 @@ <h2><dfn>disconnect()</dfn> method</h2>
     <h2>Processing</h2>
     <section data-link-for="PerformanceObserver">
     <h2>Queue a <code>PerformanceEntry</code></h2>
-      <p>To <dfn>queue a PerformanceEntry</dfn> (<var>new entry</var>) with an
-      optional <var>add to performance entry buffer</var> flag, which is unset
-      by default, run these steps:</p>
+      <p><i>Specifications</i> of performance entries may define an algorithm
+        <var>should append to observer</var> which receives a <a>PerformanceEntry</a> and
+        a <a>PerformanceObserver</a> as input and returns a boolean specifying whether the
+        observer should be notified of the entry.
+      <p>The <dfn>queue a PerformanceEntry</dfn> <var>new entry</var> with an optional
```

Comment:
I'm not following the grammar of this sentence.

Reply:
Sorry, which? The one starting with Specifications, or with 'The queue'

Reply:
Definitely The queue sentence is incorrect. The first word should be 'To'.